book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,251 @@
+
+### Ethics and law
+
+Laws have been made by humans.
+They reflect the outcomes of negotiations between different people, with different interests and different amounts of power.
+Complying with laws is usually necessary, but *rarely sufficient*, to ensure that you are acting ethically.
+
+### Domestic and regional law
+
+You should be aware of what laws cover your work.
+This will depend on which jurisdictions you work in: usually the geographical area covered by a law.
+You should be aware of the laws that cover you at local, state or regional, national, and international levels. 
+
+You should also be aware what steps you should take to ensure that you are compliant with all relevant laws. 
+This section is not exhaustive!
+It is designed to give you some idea of the kinds of laws you should be aware of.
+
+#### Data protection law
+
+Data protection laws cover what is permitted by law when collecting and processing data. 
+The aim of data protection laws is to protect the privacy of individuals when collecting and using information about them.
+
+Data protection laws need to be updated frequently in order to make sure they cover all the ways that different actors are using new forms of technology to collect and use data in new ways. 
+Because data protection laws sit at the limits of what companies - especially big tech giants - are able to do, the process of making, updating and enforcing data protection laws has become increasingly newsworthy.
+New pieces of legislation such as the General Data Protection Regulation ([GDPR](https://gdpr-info.eu/)) in the EU have attracted a lot of attention.
+
+Many countries have a national data protection authority, which is tasked with protecting data.
+They often produce guidance on how to comply with data protection laws in their country.
+Wikipedia has a [list of data protection authorities](https://en.wikipedia.org/wiki/National_data_protection_authority) covering lots of countries.
+
+Data protection laws are sometimes confused with privacy laws, but they are not the same thing. 
+
+#### Privacy law
+
+Privacy laws protect people's ability to keep themselves, or information about themselves, away from public view. 
+The specific content of privacy laws varies between jurisdictions.
+Laws may be general, or they may only cover certain kinds of information, sometimes referred to as 'sensitive information' or 'personally identifiable information.'
+This might include, for example, medical or financial information about a person.
+
+Privacy laws have periodically been updated to take into account new technologies, which have offered new ways for privacy to be invaded and for information about people to be shared. 
+This started in the 19th century when newspapers became widespread, and is still happening as legislators try to keep up with developments in the internet and in data collection.
+
+Privacy is sometimes confused with data protection, but it is much broader. 
```

Comment:
This line is really interesting. Could there be an example to clarify? I can't come up with a good one, sadly! Maybe peeking into people's homes is one?

Reply:
+1, examples of cases that help differentiate between the different entities each law set covers would be a great addition! I would not mind toy examples for the law newbie like me 😄